youtube_dl/extractor/wikimedia.py

```diff
@@ -0,0 +1,65 @@
+from .common import InfoExtractor
+from ..utils import get_element_by_class, compat_urlparse, clean_html
+import re
+
+
+class WikimediaIE(InfoExtractor):
+    _NETRC_MACHINE = 'wikimediaorg'
+    IE_NAME = 'wikimedia.org'
+    _API_BASE_URL = 'https://commons.wikimedia.org/'
+    _VALID_URL = r'https://commons.wikimedia.org/wiki/File:(?P<id>[^/]+)'
+
+    _TEST = {
+        'url': 'https://upload.wikimedia.org/wikipedia/commons/transcoded/d/d7/Die_Temperaturkurve_der_Erde_%28ZDF'
+               '%2C_Terra_X%29_720p_HD_50FPS.webm/Die_Temperaturkurve_der_Erde_%28ZDF%2C_Terra_X%29_720p_HD_50FPS'
+               '.webm.480p.vp9.webm',
+        'description': 'Deutsch:  Beschreibung auf der Seite: "Im Verlauf der Erdgeschichte glich das Klima einer '
+                       'Achterbahnfahrt. Die „Fieberkurve“ unseres Planeten zeigt die globalen Temperaturschwankungen '
+                       'bis heute – rekonstruiert anhand von historischen Klimadaten."\nZu Wikimedia Commons '
+                       'hochgeladen von: PantheraLeo1359531.\nHinweise zur Weiterverwendung: '
+                       'https://www.zdf.de/dokumentation/terra-x/terra-x-creative-commons-cc-100.html'
+                       '.\nVereinfachender Verlauf in der Geschichte der Erde, für die Zukunft spätestens ab dem Jahr '
+                       '2050 mit spekulativem Verlauf in der Prognose (ausgeprägtes Global-warming-Szenario ist '
+                       'dargestellt).English:  Climate change, Temperature in history of Earth, Video of Terra X.',
+        'ext': 'webm', 'id': 'Die_Temperaturkurve_der_Erde_(ZDF,_Terra_X)_720p_HD_50FPS.webm',
+        'title': 'Die Temperaturkurve der Erde (ZDF, Terra X) 720p HD 50FPS.webm - Wikimedia Commons',
+        'license': 'This file is licensed under the Creative Commons Attribution 4.0 International license.',
+        'author': 'ZDF/Terra X/Gruppe 5/Luise Wagner, Jonas Sichert, Andreas Hougardy', 'subtitles': {'nl': [
```

Comment:
```py
        'author': 'ZDF/Terra X/Gruppe 5/Luise Wagner, Jonas Sichert, Andreas Hougardy',
        'subtitles': {
            'nl': [{
                'ext': 'srt',
                'url': 
                     'https://commons.wikimedia.org/w/api.php?action=timedtext&lang=nl&'
                     'title=File%3ADie_Temperaturkurve_der_Erde_%28ZDF%2C_Terra_X'
                     '%29_720p_HD_50FPS.webm&trackformat=srt',
                },
            ],
        },
    }
```
Adding `,` after the last item in a list or dict means that adding a new item doesn't cause a change to the existing item.

I haven't put this through the CI Flake8 test so it might still complain ...

Reply:
I resolved the test & comma can your review it? I really appreciate your help